### DISCUSSION TITLE
Module 4 project

### DISCUSSION THREAD

**Discussion by C-Doll** [cooperrc/computational-mechanics#174] [Q&A] [answered]
I'm a bit confused about the part of the project where it is mentioned that node 4 has a -300N load. 
![image](https://user-images.githubusercontent.com/121252243/212095811-dca80581-472f-466e-b02c-b4b0ebd5304c.png)

But earlier it is mentioned there are 7 nodes labeled n1 - 7.
If we wanted a load at n=4 we would need F[3]=-300  not F[5] right? 🤔

Can anyone tell me if I am missing anything?

**Comment by C-Doll** [comment]
Moving forward I immediately run into a mismatch in array sizes when trying to solve for the forward substitution method. I suppose the correct array size for F is going to be the element size, not the node size and would make the explicit direction of F[5]=-300 the 4th node something that I should just trust.

**Comment by cooperrc** [comment, marked as answer]
The node displacements and forces are labeled , x, then y so for 7 nodes, there are 14 x-y-components, 

$$\bar{u} = [u_{1x},~u_{1y},~u_{2x},~u_{2y},~,u_{3x},~u_{3y},~,u_{4x},~u_{4y},~,u_{5x},~u_{5y}~,u_{6x},~u_{6y},~,u_{7x},~u_{7y}]$$

and


$$\bar{F} = [F_{1x},~F_{1y},~F_{2x},~F_{2y},~,F_{3x},~F_{3y},~,F_{4x},~F_{4y},~,F_{5x},~F_{5y}~,F_{6x},~F_{6y},~,F_{7x},~F_{7y}]$$

**Reply by C-Doll** [reply]
Ah, that makes much more sense, thank you.
